addons/wowl/static/src/components/dialog/dialog.ts

```diff
@@ -4,30 +4,35 @@ const { useRef, useExternalListener } = hooks;
 const { Portal } = misc;
 
 interface DialogProps {
+  contentClass: String;
   fullscreen: boolean;
   renderFooter: boolean;
   renderHeader: boolean;
   size: "modal-xl" | "modal-lg" | "modal-sm";
+  technical: Boolean;
   title: String;
 }
 
 export class Dialog extends Component<DialogProps, OdooEnv> {
   static components = { Portal };
   static props = {
+    contentClass: String,
     fullscreen: Boolean,
     renderFooter: Boolean,
     renderHeader: Boolean,
     size: {
       type: String,
       validate: (s: string) => ["modal-xl", "modal-lg", "modal-sm"].includes(s),
     },
+    technical: Boolean,
     title: String,
   };
   static defaultProps = {
     fullscreen: false,
     renderFooter: true,
     renderHeader: true,
     size: "modal-lg",
+    technical: true,
```

Comment:
technical by default?

Reply:
We use the same logic as in web/owl_dialog